src/client/http.ts

```
@@ -75,6 +76,19 @@ export type RetryConfig =
        * ```
        */
       backoff?: (retryCount: number) => number;
```

From the comments, I couldn't understand the usecase for the normal backoff. I know ratelimit is applied for 429, but what is this for? Maybe we can explain a bit more in the docstring

---

it's for when fetch throws an error, instead of returning a response with some status (200 or non-200)

---

Oh ok, after reading the code I understood, a little note in the comments would be very helpfull

---

updated docs 👍